### ISSUE TITLE
PurifyCSS Whitelist Not Working Properly

### ISSUE THREAD

**Issue by aaronhuisinga** [JeffreyWay/laravel-mix#919]
- Laravel Mix Version: 1.0.3 (`npm list --depth=0`)
- Node Version (`node -v`): 8.1.2
- NPM Version (`npm -v`): 5.0.3
- OS: Ubuntu 16.04 & MacOS Sierra

### Description:
When PurifyCSS is enabled and a whitelist is defined within the options method in the Mix config, elements in the whitelist are still being removed. My config is the same as before 1.0.0, and it still works fine there if I rollback the Mix version.

### Steps To Reproduce:
1. Install Laravel & Laravel Mix 1.0.0+
2. Define a stylesheet, and enable PurifyCSS
3. Add whitelist for PurifyCSS to options
4. Run Mix and check if whitelisted tags exist

### Example Mixfile:
```js
const {mix} = require('laravel-mix');
let webpack = require('webpack');

mix.options({
        purifyCss: {
            purifyOptions: {
                whitelist: ['*fa-*', '*redactor-*', '*twitter-typeahead*']
            }
        }
    })
    .sass('resources/sass/app/app.scss', 'public/assets/css/app.css');
```

**Comment by aaronhuisinga** [comment]
I've attempted moving options to the top and bottom of the config file, and have checked commits for changes to the purifyCSS options. This looks to have been caused by some internal change in v1.0.0+

**Comment by JeffreyWay** [comment]
Thanks, @aaronhuisinga! This is fixed in 1.0.4. 

https://github.com/JeffreyWay/laravel-mix/releases/tag/v1.0.4
